doc/devel/documenting_mpl.rst

```diff
@@ -19,6 +24,32 @@ The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/m
 6. pillow
 7. graphviz
 
+.. note::
+
+  * You'll need a minimal working latex distribution for many examples to run.
+  * `Graphviz <http://graphviz.readthedocs.io/en/latest/manual.html>`_ is not pip-installable so you need
+    to install this on your own. This is straightforward with something like
```

Comment:
Not sure the conda reference is helpful here 'cause it's not really contextualized.